zazu/repo/commands.py

```diff
@@ -67,12 +69,17 @@ def clone(ctx, repository_url, nohooks):
         --nohooks: does not install git hooks in the cloned repo
     """
     try:
-        git.Git().clone(repository_url)
+        repo = git.Repo.clone_from(repository_url, '{}/{}'.format(os.getcwd(), repository_url.rsplit('/', 1)[-1].replace('.git', '')))
```

Comment:
I'd split this into 2 lines for readability. you can make a var that represents the destination_path and then pass it ti clone_from.